build.zig

```diff
@@ -27,13 +27,11 @@ pub fn build(b: *zbs.Builder) !void {
 
     const examples = b.option(bool, "examples", "Set to true to build examples") orelse false;
 
-    // Sigh, why are the conventions inconsistent like this.
-    const resolved_prefix = try std.fs.path.resolve(b.allocator, &[_][]const u8{b.install_prefix.?});
-    if (std.mem.eql(u8, resolved_prefix, "/usr")) {
-        b.installFile("example/init", "../etc/river/init");
-    } else {
-        b.installFile("example/init", "etc/river/init");
-    }
+    const default_config_path = try std.fs.path.resolve(
+        b.allocator,
+        &[_][]const u8{ b.install_prefix.?, "etc/river/init" },
```

Comment:
We still need the logic to install to `/etc/river/init` if the prefix is `/usr`.

Reply:
Changed the commit message. What is the purpose of moving out of `/usr` if river can find it there via `default_config_path`?